OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -92,7 +92,26 @@ public function __construct(HttpClientInterface $httpClient, HttpUtils $httpUtil
 
         // Resolve merged options
         $resolver = new OptionsResolver();
+
         $this->configureOptions($resolver);
+
+        // See Symfony\Component\Form\ResolvedFormType logic
+        $baseClasses = array(
+            'HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\AbstractResourceOwner',
+            'HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\GenericOAuth1ResourceOwner',
+            'HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\GenericOAuth2ResourceOwner',
```

Comment:
IMO, the generic owners don't need to be in this list. They don't need a special treatment here as they don't need to overwrite the old method from AbstractResourceOwner